src/transport/SessionManager.cpp

```diff
@@ -147,6 +147,9 @@ CHIP_ERROR SessionManager::PrepareMessage(const SessionHandle & sessionHandle, P
 {
     MATTER_TRACE_SCOPE("PrepareMessage", "SessionManager");
 
+    // FIXME: Fill it up
```

Comment:
Is this comment old? It looks like this does get filled below.

Reply:
ooops ... fixed and moved closer to initialization. Good catch!